plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -357,8 +361,8 @@ func (s *serviceAccount) mountServiceAccountToken(serviceAccount *api.ServiceAcc
 		// We don't have an API token to mount, so return
 		if s.RequireAPIToken {
 			// If a token is required, this is considered an error
-			// TODO: convert to a ServerTimeout error (or other error that sends a Retry-After header)
-			return fmt.Errorf("no API token found for service account %s/%s, retry after the token is automatically created and added to the service account", serviceAccount.Namespace, serviceAccount.Name)
+			return errors.NewServerTimeout(unversioned.GroupResource{Resource: "serviceaccounts"}, "create pod", 1)
```

Comment:
e.g. can we override the default timeout message `NewServerTimeout` sets up with the `no API token found for service account %s/%s, retry after the token is automatically created and added to the service account` one?